crates/adapters/src/integrated/delta_table/input.rs

```diff
@@ -960,6 +1002,9 @@ impl DeltaTableInputEndpointInner {
     ) {
         wait_running(receiver).await;
 
+        // Limit the number of connectors simultaneously reading from Delta Lake.
```

Comment:
is it released when the value is dropped?

Reply:
yes